cmd/objectstore_delete.go

```diff
@@ -0,0 +1,98 @@
+package cmd
```

Comment:
I'm stopping my review here, the above comments need repeating throughout before it's worth me reviewing it all...

1. Capitalise Object Store properly, it's a product name
2. Check the commands and output matches what Dinesh and I put together